app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,25 @@
+class PasswordResetsController < ApplicationController
+  def show
+    user = User.where(token: params[:id]).first
```

Comment:
For fetching a single record, using `find_by` is more efficient.  Check notes at the bottom of this page - https://www.gotealeaf.com/lessons/a3a6fbb9/assignments/1566